iwyu_include_picker.h

```diff
@@ -129,7 +129,14 @@ class IncludePicker {
   // important (which is why we return a vector, not a set): all else
   // being equal, the first element of the vector is the "best" (or
   // most standard) header for the symbol.
-  vector<string> GetCandidateHeadersForSymbol(const string& symbol) const;
+  vector<MappedInclude> GetCandidateHeadersForSymbol(
+      const string& symbol) const;
+
+  // As above, but given a specific including header it is possible to convert
+  // MAppedIncludes to quoted include strings (because we can for example know
```

Comment:
`MAppedIncludes` -> `mapped includes`? Or at least fix the casing :)

Reply:
Done.